### DISCUSSION TITLE
Loss of performance

### DISCUSSION THREAD

**Discussion by ghost** [hyperium/hyper#2659] [Q&A] [answered]
Hi,

According to this site,
https://www.techempower.com/benchmarks/#section=data-r20&hw=ph&test=plaintext

Hyper had a loss of performance. Not only on ranking but raw performance as well.
I am not taking any db variants into consideration, just raw file, so raw perf.

What is wrong?

Thanks

**Comment by seanmonstar** [comment, marked as answer]
The dip in performance matches up with when Rust changed it's default allocator from jemalloc to the system allocator, which is usually less performant. I imagine that isn't a coincidence.
